spec/ffg.tla

```diff
@@ -8,7 +8,15 @@ CONSTANTS
      *
      * @type: ($signedVoteMessage) => Bool;
      *)
-    VERIFY_VOTE_SIGNATURE(_)
+    VERIFY_VOTE_SIGNATURE(_),
+
+    (*
+     * TODO
```

Comment:
Do we know what `get_validator_set_for_slot` is supposed to return? From the name and the types, it looks like this function is supposed to return something like voting power. Perhaps, @saltiniroberto could help us here